README.md

```diff
@@ -9,6 +9,16 @@ nudoku is a ncurses based sudoku game.
 #### Arch Linux ####
 I heard rumors of nudoku being in the [AUR](https://aur.archlinux.org/packages/nudoku-git/).
 
+#### EL 7 (CentOS/Oracle Linux/Scientific Linux/RedHat####
+Since nudoku is in the EPEL repository, you yust need to enable the EPEL repository and run:
+
+  # yum install nudoku
```

Comment:
This one doesn't get the right formatting could you remove the # and add three times a ` obove and under the line? See in the rest of the README for examples :)